### ISSUE TITLE
Add a license

### ISSUE THREAD

**Issue by picsi** [wayspurrchen/tooter.today#1]
Ideally file headers, but a LICENSE.md file would be welcome too. :)

**Comment by wayspurrchen** [comment]
Agreed, thank you for the nudge! I'm planning to do it under MIT, I just need to reuse the styles since I used some of the Mastodon styles which I believe are AGPL.
